dashboard/test/ui/runner.rb

```diff
@@ -284,14 +284,23 @@ def browser_name_or_unknown(browser)
   browser['name'] || 'UnknownBrowser'
 end
 
-def flakiness_for_browser_feature(browser_feature)
-  TestFlakiness.test_flakiness[test_run_identifier(browser_feature[0], browser_feature[1])] || 1.0
+$calculate_flakiness = true
+# Retrieves / calculates flakiness for given test run identifier, giving up for
+# the rest of this script execution if an error occurs during calculation.
+# returns the flakiness from 0.0 to 1.0 or nil if flakiness is unknown
```

Comment:
The documented return value (0.0..1.0 or nil) does not capture the possible `false` return value if we've decided to skip calculating flakiness.  Also, three possible return types seems like a lot for this function.  Any reason not to return nil when we skip?